### DISCUSSION TITLE
how to clean the top dirty corners of an image?

### DISCUSSION THREAD

**Discussion by srini8080** [ImageMagick/ImageMagick#6382] [Digital Image Processing]
Linux
IM7

![x5](https://github.com/ImageMagick/ImageMagick/assets/12046302/4d68ea34-3909-44fd-aab0-5ef40af9d255)
 I want to clean the corners and edges of a dirty page

Thanks

**Comment by fmw42** [comment]
I do not know if Imagemagick or simple image processing would be able to help.  You might consider using tesseract to read the text and redraw it on a clean white canvas.
